### PR TITLE
[Ameba] Update ameba SDK to expose xPortResetHeapMinimumEverFreeHeapSize API

### PR DESCRIPTION
**Pull request by pankore**
#### Problem
Add reset-watermarks and read thread-metrics function.

#### Change overview

- Update chip-build-ameba/Dockerfile
- Update version
- Exposed xPortResetHeapMinimumEverFreeHeapSize FreeRTOS API to use for #19147

#### Testing
Tested docker build

### CODE REVIEW

**Comment by pankore** [comment]
This PR updates platform's sdk and corelates with #19147 .

File: integrations/docker/images/chip-build/version [hunk 1]
```diff
@@ -1 +1 @@
-0.5.78 Version bump reason: [Tizen] Fix environment variable order, add missing tizen dependencies
+0.5.79 Version bump reason: [Ameba] Add reset-watermarks and read thread-metrics function
```
Comment (andy31415) [inline integrations/docker/images/chip-build/version:1]: why is a dockerfile change required for this ?

I thought these items are implemented in chip source code.

**Review by bzbarsky-apple — APPROVED** [review]

File: integrations/docker/images/chip-build/version (hunk 1 above)
Comment (pankore) [reply integrations/docker/images/chip-build/version:1]: This update meant to adapt the 'new functions added in chip source code' in the platform sdk . The description is not accurate, I will change it. Thanks

**Review by Damian-Nordic — APPROVED** [review]

**Review by andy31415 — APPROVED** [review]

**Review by andy31415 — COMMENTED** [review]
I approved this, however I believe the description is still incorrect/unclear.

This clearly grabs an updated SDK for Ameba. The description should say why this new SDK is pulled and what the SDK contains.

When I read "adapt ...." I expect code changes to adapt something. I think this should rather be "pull SDK so that it exposes ...."

File: integrations/docker/images/chip-build/version [hunk 2]
```diff
@@ -1 +1 @@
-0.5.78 Version bump reason: [Tizen] Fix environment variable order, add missing tizen dependencies
+0.5.79 Version bump reason: [Ameba] To adapt the 'reset-watermarks' and 'read thread-metrics' functions added in Matter sdk
```
Comment (andy31415) [inline integrations/docker/images/chip-build/version:1]: what does "to adapt" mean? 

Could this be for #19147 ? In which case I assume this should rather be "update ameba sdk to expose xPortResetHeapMinimumEverFreeHeapSize freeRTOS API" or something similar.

Comment (pankore) [reply integrations/docker/images/chip-build/version:1]: I have changed the description.

**Review by woody-apple — APPROVED** [review]
Fast tracking docker updates

**Comment by pankore** [comment]
Hi @andy31415 ,
Please kindly help to review this PR again, the failed cases should not relate to the commits. Thanks!

**Pull request merged**